force-app/main/default/classes/DuplicateEnrollment.cls

```diff
@@ -0,0 +1,21 @@
+// Helper class
+public with sharing class DuplicateEnrollment {
```

Comment:
A better name for the class would be DuplicateEnrollmentPreventor.

https://www.geeksforgeeks.org/java-naming-conventions/